libraries/AP_BattMonitor/AP_BattMonitor_Logging.cpp

```diff
@@ -0,0 +1,70 @@
+#include "AP_BattMonitor_Backend.h"
+#include <AP_Logger/AP_Logger.h>
+
+extern const AP_HAL::HAL& hal;
+
+// Write BAT data packet(s)
+void AP_BattMonitor_Backend::Log_Write_BAT(const uint8_t instance, const uint64_t time_us) const
+{
+    bool has_temp = (AP_HAL::millis() - _state.temperature_time) <= AP_BATT_MONITOR_TIMEOUT;
+    bool has_current1 = has_current();
```

Comment:
```suggestion
    bool has_current = has_current();
```

Not sure why this has a `current1`? Seems less readable.

Reply:
Fixed it. Renamed it so it didnt conflict with function name.